saleor/account/emails.py

```diff
@@ -9,14 +9,14 @@
 
 @shared_task
 def send_password_reset_email(context, recipient):
-    reset_url = build_absolute_uri(
+    url = build_absolute_uri(
         reverse(
             'account:reset-password-confirm',
             kwargs={
                 'uidb64': context['uid'],
                 'token': context['token']}))
     context = get_email_base_context()
-    context['reset_url'] = reset_url
+    context['url'] = url
```

Comment:
Could you keep meaningful variable names everywhere for the URLs? (like it was before, `reset_url`, `password_set_url`)